python/rmm/_cuda/stream.pxd

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from libc.stdint cimport uintptr_t
+from libcpp cimport bool
+
+from rmm._lib.cuda_stream_view cimport cuda_stream_view
+from rmm._lib.lib cimport cudaStream_t
+
+cdef class Stream:
+    cdef cudaStream_t _ptr
```

Comment:
instead of this being `_ptr` should we make this something like `_cudastream_t`?

Reply:
Yes but here are some alternatives for your perusal: 

`_cuda_stream`
`_cuda_stream_t`
`_stream`
`c_obj`

Reply:
I think `_stream` or `c_obj` would be a bit ambiguous. Fine with any of the others though 😄